Plugins/Flow.Launcher.Plugin.Program/Main.cs

```diff
@@ -209,13 +212,11 @@ private void DisableProgram(IProgram programToDelete)
                 return;
 
             if (_uwps.Any(x => x.UniqueIdentifier == programToDelete.UniqueIdentifier))
-                _uwps.Where(x => x.UniqueIdentifier == programToDelete.UniqueIdentifier)
-                    .FirstOrDefault()
+                _uwps.FirstOrDefault(x => x.UniqueIdentifier == programToDelete.UniqueIdentifier)!
```

Comment:
added a '!' is this logic correct?

Reply:
removed '!'.

Reply:
The! Means it will not be null. It is not logical operator. Though in this case probably we should change the check.